src/util/helpers.py

```diff
@@ -317,25 +317,38 @@ def resolve_model_fields_path(cr, model, path):
     return [FieldsPathPart(**row) for row in cr.dictfetchall()]
 
 
-def _remove_export_lines(cr, model, field=None):
-    q = """
-    SELECT el.id,
-           e.resource,
-           STRING_TO_ARRAY(el.name, '/')
-      FROM ir_exports_line el
-      JOIN ir_exports e
-        ON el.export_id = e.id
-    """
+def _remove_import_export_paths(cr, model, field=None):
+    export_q = """
+            SELECT el.id,
+                   e.resource,
+                   STRING_TO_ARRAY(el.name, '/')
+              FROM ir_exports_line el
+              JOIN ir_exports e
+                ON el.export_id = e.id
+        """
     if field:
-        q = cr.mogrify(q + " WHERE el.name ~ %s ", [r"\y{}\y".format(field)]).decode()
-    cr.execute(q)
-    to_rem = [
-        line_id
-        for line_id, line_model, line_path in cr.fetchall()
-        if any(
-            x.field_model == model and (field is None or x.field_name == field)
-            for x in resolve_model_fields_path(cr, line_model, line_path)
-        )
-    ]
-    if to_rem:
-        cr.execute("DELETE FROM ir_exports_line WHERE id IN %s", [tuple(to_rem)])
+        export_q = cr.mogrify(export_q + " WHERE el.name ~ %s ", [r"\y{}\y".format(field)]).decode()
+
+    import_q = """
+        SELECT id,
+               res_model,
+               STRING_TO_ARRAY(field_name, '/')
+          FROM base_import_mapping
+        """
+    if field:
+        import_q = cr.mogrify(import_q + " WHERE field_name ~ %s ", [r"\y{}\y".format(field)]).decode()
+
+    for query, impex_model in [(export_q, "ir.exports.line"), (import_q, "base_import.mapping")]:
+        cr.execute(query)
+        to_rem = [
+            path_id
+            for path_id, related_model, path in cr.fetchall()
+            if any(
+                x.field_model == model and (field is None or x.field_name == field)
+                for x in resolve_model_fields_path(cr, related_model, path)
+            )
+        ]
+        if to_rem:
+            from .records import remove_records
```

Comment:
I'm not a fan of this import in a function body, moreover in a loop.
Maybe this function should be moved to `records.py`

Reply:
I don't see any issue with the new dependency. We can figure out something later if it becomes an issue.

Reply:
> is that an ok consequence?

Yes. I don't see why it can be a problem.